src/state/component-repository/components/actions.js

```diff
@@ -212,9 +212,16 @@ export const pollECRComponentInstallStatus = (componentCode, stepFunction) => di
           dispatch(finishComponentInstallation(componentCode, res.payload));
         } else {
           dispatch(componentInstallationFailed(componentCode));
-          if (res.payload.status === ECR_COMPONENT_INSTALLATION_STATUS_ROLLBACK) {
+          if (res.payload.installErrorMessage) {
             dispatch(addToast(
-              { id: 'componentRepository.components.installRollback' },
+              res.payload.installErrorMessage,
+              TOAST_WARNING,
+            ));
+          }
+
+          if (res.payload.rollbackErrorMessage) {
+            dispatch(addToast(
+              res.payload.rollbackErrorMessage,
               TOAST_WARNING,
```

Comment:
does toast show correctly the message here?

Reply:
it does! you can see the screenshot attached in this PR. The error message will come directly from the BE now.